modules/rte_mis_school/rte_mis_school.module

```diff
@@ -783,13 +818,13 @@ function rte_mis_school_views_query_alter(ViewExecutable $view, QueryPluginBase
     if (array_intersect($roles, ['district_admin', 'block_admin'])) {
       $user = \Drupal::entityTypeManager()->getStorage('user')->load(\Drupal::currentUser()->id());
       if ($user instanceof UserInterface) {
+        $id = [0];
```

Comment:
Didn't get this?

Reply:
Can't we have an empty array simply?

Reply:
No, this is for exceptional case where location is not assigned to block and district admin. it will throw a WSOD error or If we add `!empty` check then the admin without location will have all the listing.

Reply:
Cool